lib/checkPullRequestLabels.js

```diff
@@ -193,12 +193,13 @@ module.exports.checkChangelogLabel = async function (context) {
   );
 
   const userName = pullRequest.user.login;
-  const changelogLabel = getChangelogLabelFromPullRequest(pullRequest);
+  const changelogLabel = await getChangelogLabelFromPullRequest(pullRequest);
```

Comment:
Why is await added here?

Reply:
Sir previously without await `getChangelogLabelFromPullRequest(pullRequest);` returns `undefined` to `changeloglabel` because of which `Boolean(changelogLabel);` return false to `hasChangelogLabel` so to maintain the synchronous execution i used await.

Reply:
But `getChangelogLabelFromPullRequest` does not return a promise. So await doesn't do anything there.